setup.py

```diff
@@ -21,7 +21,7 @@
       author_email='support@confluent.io',
       url='https://github.com/confluentinc/confluent-kafka-python',
       ext_modules=[module],
-      packages=find_packages(exclude=("tests",)),
+      packages=find_packages(exclude=("tests","tests.*")),
```

Comment:
Please separate two strings with a space to fix the flake warning failing the build.

Reply:
:facepalm:

Sorry about that